unity-client/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -480,6 +480,7 @@ public static Bounds BuildMergedBounds(Renderer[] renderers)
             return bounds;
         }
 
+
```

Comment:
No big deal but watchout in the future so we dont polute the git history with meaningless changes!